tests/machine/test_machine.py

```diff
@@ -126,3 +126,8 @@ def test_get_machine_schema_types_return_mtype(mocked_types, mocked_machines):
         "stat__test_float",
         "stat__test_string",
     ]
+
+
+def test_get_machines_v1(get_client):
```

Comment:
Add a note that is tested against demo and if that env is changed this test has to as well

Reply:
Done